doc/administrator/upgrades/index.md

```diff
@@ -11,7 +11,7 @@ description: "Upgrade Notes"
 
 The majority of IRIDA's upgrade notes can be seen at <https://github.com/phac-nml/irida/blob/master/UPGRADING.md>.  When there are more major upgrades to the IRIDA system, database, deployment, etc. this page will go further in depth about how to properly back up your system, perform the upgrade, and recover from problems.
 
-# 19.04
+# 19.05
 ## FastQC translation to filesystem
```

Comment:
I wonder if we should also include in the **In case of error** section of this document both that you will likely be unable to access IRIDA in case of an error, and if you try to re-run Tomcat again you may encounter messages like below and will have to search previous messages in the log file to track down the first database upgrade error?

```
Change Set classpath:ca/corefacility/bioinform
atics/irida/database/changesets/19.05/fastqc-filesystem.xml::fastqc-filesystem::tom failed.  Error: Duplicate column name 'analysis_id' [Failed SQL: ALTER TABLE irida_test.analysis_output_file ADD analysis_id BIGINT NULL]
```

This could be added as another pull request (since it's unrelated to switching the version).

Reply:
I'd say lets do this in a later PR because it will apply to more than just this upgrade.  It along with the "not supported if you use development versions of IRIDA" thing need to be linked from lots of places.

Also more directly on this comment, I think it's pretty clear that you wouldn't be able to run IRIDA in case of a DB upgrade issue.  We have lots of mitigation information for this particular update.

Reply:
Sounds good :+1: